service-api/src/main/java/greencity/dto/breaktime/BreakTimeDto.java

```diff
@@ -0,0 +1,24 @@
+package greencity.dto.breaktime;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.annotations.ApiModelProperty;
+import java.time.LocalTime;
+import javax.validation.constraints.NotNull;
+import lombok.*;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode(exclude = {"startTime", "endTime"})
+public class BreakTimeDto {
+    @NotNull
+    @JsonFormat(pattern = "HH:mm", shape = JsonFormat.Shape.STRING)
```

Comment:
Those two annotation should be on the bean accepted by controller.

Reply:
removed